.travis.yml

```diff
@@ -21,10 +21,10 @@ install:
   - conda info -a
 
   # create environment and install dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy=1.11.2 scipy nose pip matplotlib coverage
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION coveralls
   - source activate test-environment
-  - pip install coveralls
-  - python setup.py install
+  - conda env update -n test-environment -f environment.yml
+  - pip install .
```

Comment:
Why pip instead of setup.py?

Reply:
Because `pip` is recommended (eg. https://stackoverflow.com/questions/15724093/difference-between-python-setup-py-install-and-pip-install). It keeps track of where stuff is installed so it can be later uninstalled if necessary.